frontend/src/container/LogControls/index.tsx

```diff
@@ -56,24 +58,43 @@ function LogControls(): JSX.Element | null {
 	}
 	return (
 		<Container>
-			<Button size="small" type="link" onClick={handleGoToLatest}>
+			<Button
+				loading={isLoading || isLoadingAggregate}
+				disabled={isLoading || isLoadingAggregate}
+				size="small"
+				type="link"
+				onClick={handleGoToLatest}
+			>
 				<FastBackwardOutlined /> Go to latest
 			</Button>
 			<Divider type="vertical" />
-			<Button size="small" type="link" onClick={handleNavigatePrevious}>
+			<Button
+				loading={isLoading || isLoadingAggregate}
+				disabled={isLoading || isLoadingAggregate}
+				size="small"
+				type="link"
+				onClick={handleNavigatePrevious}
+			>
 				<LeftOutlined /> Previous
 			</Button>
-			<Button size="small" type="link" onClick={handleNavigateNext}>
+			<Button
+				loading={isLoading || isLoadingAggregate}
+				disabled={isLoading || isLoadingAggregate}
+				size="small"
+				type="link"
+				onClick={handleNavigateNext}
+			>
 				Next <RightOutlined />
 			</Button>
 			<Select
-				style={{ width: 120 }}
+				loading={isLoading || isLoadingAggregate}
+				disabled={isLoading || isLoadingAggregate}
```

Comment:
When loading is true the button would be disabled, maybe we can remove this

Reply:
yup this one is updated